cluster/local/util.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
```

Comment:
I think cluster/local is going to be confusing vs hack/local-up-cluster.sh (which does NOT use kind and predates it)

Reply:
I see. The e2e test local debugging script limited the provider(gce, azure, ..., local, with `kind` missing) somewhere. So I just use `local` to provide a working version for further discussion on the fixing idea, which is happening now. I'll take a further look to resolve this.